docs/examples/example_autosave_ioc.py

```diff
@@ -0,0 +1,34 @@
+from softioc import autosave, builder, softioc
+import cothread
+
+# Set the record prefix
+builder.SetDeviceName("MY-DEVICE-PREFIX")
+
+# Create records, set some of them to autosave, also save some of their fields
+
+builder.aOut("AO", autosave=True)
+builder.aIn("AI", autosave_fields=["PREC", "EGU"])
+builder.boolIn("BO")
+builder.WaveformIn("WAVEFORMOUT", [0, 0, 0, 0], autosave=True)
+with autosave.Autosave(True, ["LOPR", "HOPR"]):
```

Comment:
I wonder about the `True` argument here, it seems a bit redundant pretty well all of the time.  Howe about `Autosave` taking a `*args` list and defaulting to `"VAL"` if nothing is set?  Would that cover the normal bases and be sensible?  Am thinking you could code it something like:
```python
class Autosave:
    def __init__(self, *autosave_fields):
        if not autosave_fields: autosave_fields = ["VAL"]
        ...
```
Does mean people would have to write `"VAL"` sometimes which might be a surprise.